### PR TITLE
[Wicked] Add IPIP tunnel test case

### PR DESCRIPTION
**Pull request by jlausuch**
- Related ticket: https://progress.opensuse.org/issues/36202
- Verification runs: 
http://fromm.arch.suse.de/tests/742
http://fromm.arch.suse.de/tests/743

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi

File: lib/wickedbase.pm [hunk 1]
```diff
@@ -43,6 +43,12 @@ sub get_ip {
     elsif ($args{type} eq 'sit1') {
         return $args{is_wicked_ref} ? '2001:0db8:1234::000e' : '2001:0db8:1234::000f';
     }
+    elsif ($args{type} eq 'tunl1') {
+        return $args{is_wicked_ref} ? '3.3.3.10' : '3.3.3.11';
+    }
+    elsif ($args{type} eq 'br0') {
+        return $args{is_wicked_ref} ? '10.0.2.10' : '10.0.2.11';
```
Comment (asmorodskyi) [inline lib/wickedbase.pm]: isn't this suppose to be part of another commit?

Comment (jlausuch) [reply lib/wickedbase.pm]: yes, sorry, let's merge https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5396  first and then I'll rebase this on master.

File: tests/wicked/advanced_ref.pm [hunk 2]
```diff
@@ -33,22 +33,31 @@ sub create_tunnel_with_commands {
...
     record_info('Test 3', 'Create a SIT interface with IP commands');
-    $self->create_tunnel_with_commands("sit1", "127");
-    # Lock until parent creates mutex 'test_1_ready'
+    $self->create_tunnel_with_commands("sit1", "sit", "127");
+    # Lock until parent creates mutex 'test_3_ready'
     mutex_lock('test_3_ready');
     # Unlock mutex to end test
     mutex_unlock('test_3_ready');
 
+    record_info('Test 6', 'Create a SIT interface with IP commands');
```
Comment (asmorodskyi) [inline tests/wicked/advanced_ref.pm]: why 6? there is another PR with 4th but did I missed 5th somewhere?

Comment (jlausuch) [reply tests/wicked/advanced_ref.pm]: I am trying to keep the "legacy ifcfg"-type tests in even numbers, so the xml cases go next. Example, test4 - bridge with ifcfg. Test5: bridge with xml.  and so on.

File: lib/wickedbase.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/wickedbase.pm]: both PRs were not merged yet, better delete code from here and add it to #5396. so ALL code related to bridge will be in one commit

Comment (jlausuch) [reply lib/wickedbase.pm]: done. Anyway, I might have to solve a git conflict
